src/parser/mod.rs

```diff
@@ -15058,6 +15069,57 @@ impl<'a> Parser<'a> {
         }
     }
 
+    /// Parse [Statement::Go]
+    fn parse_go(&mut self) -> Result<Statement, ParserError> {
+        // previous token should be a newline (skipping non-newline whitespace)
+        // see also, `previous_token`
+        let mut look_back_count = 2;
+        loop {
```

Comment:
This can maybe be simplified. Also, needs a test when `go` starts a file but followed by other statements